scripts/create-container-registry-creds.sh

```diff
@@ -13,7 +14,9 @@ if [[ "${CI:-}" == "true" ]]; then
     CTR_REGISTRY_PASSWORD=${DOCKER_PASS:-}
 fi
 
-[[ -z "$CTR_REGISTRY_PASSWORD" ]] && exit 0
+if [[ "$USE_PRIVATE_REGISTRY" = false ]]; then
```

Comment:
Since we have this line, can you remove where we use this variable to optionally call the script in `run-osm-demo.sh`

We'll just let the check here handle it